testutil/apiload/sample-apiload.conf

```diff
@@ -0,0 +1,15 @@
+; The path to the dcrdata server being tested.
+; server=http://localhost:7777
```

Comment:
Note here that trailing slash must be omitted (or strip it in config validation).